app/views/organisations/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Departments, agencies and public bodies - GOV.UK" %>
+
+<h1>Departments, agencies and public bodies - GOV.UK</h1>
+<br>
```

Comment:
Is this extra spacing needed? If yes, can we do this with CSS instead? (we should avoid using HTML tags for styling)